web/src/i18n/translations/ja.yml

```
@@ -229,7 +229,7 @@ Multi line text: 複数行テキスト
 Markdown text: マークダウン
 Rich text which supports md style: マークダウン対応のリッチテキスト
 Asset file: アセット
-true/false field: ''
+true/false field: true/false フィールド
```

```suggestion
true/false field: 真偽
```

Note: In Notion, "checkbox" is used.

---

Done!